distribution/binary/BUILD

```diff
@@ -0,0 +1,42 @@
+load("@envoy//bazel:envoy_build_system.bzl", "envoy_package")
+load("@rules_pkg//pkg:mappings.bzl", "pkg_files")
+load("@rules_pkg//pkg:pkg.bzl", "pkg_tar")
+load("@envoy//distribution/binary:compiler.bzl", "bundled")
+
+licenses(["notice"])  # Apache 2
+
+envoy_package()
+
+bundled(
+    name = "envoy",
+    mode = "opt",
+    visibility = ["//visibility:public"],
+    stripopts = ["--strip-all"],
```

Comment:
yeah this is why I am unsure if the dbg will work but per our chat if we put this in 1.27 then it wont block edge